app.rb

```diff
@@ -0,0 +1,85 @@
+### Arrays
+
+# Starting with the following array...
+
+
+planeteers = ["Earth", "Wind", "Captain Planet", "Fire", "Water"]
+
+
+# Access the second value in `planeteers`.
+
+
+planeteers[1]
+
+
+# Add "Heart" to the end of `planeteers`.
+
+
+planeteers.push("Heart")
+
+
+# Remove "Captain Planet" from the array (without using a method).
+
+
+
+# Combine `planeteers` with `rangers = ["Red", "Blue", "Pink", "Yellow", "Black"]` and save the result in a variable called `heroes`.
+
+
+heroes = planeteers + rangers
+
+
+# Alphabetize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+heroes.sort_by(&:downcase)
+
+# Randomize the contents of `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+heroes.shuffle
+
+#### Bonus
+
+# Select a random element from `heroes` using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+
+
+# Select all elements in `heroes` that begin with "B" using a method. [The Ruby documentation might help](http://ruby-doc.org/core-2.6.1/Array.html).
+
+result = heroes.select {|word| word.include?"B"}
```

Comment:
Great way of solving this problem but using include will return any word that has "B" character in it , we want to check the first character so we can use

```rb
heroes.select{|heros| heros[0] === 'B'}
// or
heroes.select{|heros| heros.start_with? 'B'}
```